WildAlertApi/WildAlert.UnitTests/GetSensorsQueryHandlerTests.cs

```diff
@@ -40,6 +40,6 @@ public async Task Handle_ShouldReturnAllSensors()
         // Act
         var result = await sut.Handle(query, CancellationToken.None);
         // Assert
-        result.Should().BeEquivalentTo(_context.Sensors);
+        result.Count().Should().Be(5);
```

Comment:
5 is kinda a "magic" number here. Should it be something `_context.Sensors.ToList().Count`. And even better
`result.Should().HaveSameCount(_context.Sensors)`